CONTRIBUTING.md

```diff
@@ -37,23 +31,16 @@ Make sure everything is correctly setup with:
 npm test
 ```
 
-## Packages
+Linting and prettification is performed automatically on `git push`. However you should run them before any `git commit`
+using:
 
-This is a monorepo. Below are a list of packages included.
+```bash
+npm run format
+```
 
-<!-- AUTO-GENERATED-CONTENT:START (PACKAGES) -->
+Alternatively you can setup your IDE to integrate with Prettier and ESLint for JavaScript and Markdown files.
 
-- [@netlify/build](./packages/build) Netlify build module [npm link](https://www.npmjs.com/package/@netlify/build).
-- [@netlify/cache-utils](./packages/cache-utils) Utility for caching files in Netlify Build
-  [npm link](https://www.npmjs.com/package/@netlify/cache-utils).
-- [@netlify/config](./packages/config) Netlify config module [npm link](https://www.npmjs.com/package/@netlify/config).
-- [@netlify/functions-utils](./packages/functions-utils) Utility for adding Functions files in Netlify Build
-  [npm link](https://www.npmjs.com/package/@netlify/functions-utils).
-- [@netlify/git-utils](./packages/git-utils) Utility for dealing with modified, created, deleted files since a git
-  commit [npm link](https://www.npmjs.com/package/@netlify/git-utils).
-- [@netlify/run-utils](./packages/run-utils) Utility for running commands inside Netlify Build
-  [npm link](https://www.npmjs.com/package/@netlify/run-utils).
-  <!-- AUTO-GENERATED-CONTENT:END (PACKAGES) -->
+This is a monorepo. Below are a list of packages [here](packages).
```

Comment:
Editing to make a little more sense now that the list is gone.

```suggestion
This is a monorepo. You can find the included packages in the [packages](packages) directory.
```

Reply:
Oh that was completely improper grammar there. :fr: